### PR TITLE
MAGECLOUD-2128 Multi-site URLs

### PR DESCRIPTION
**Pull request by hguthrie**
These are updates to the configs for multi-sites.
- removed old note about changing `.` to `---`
- updated PHP code in steps
- cleaned two config guide topics
- deleted outdated screenshots and replaced with careful UI descriptions about Base URL, Base Link URL, and repeating for the Base URL Secure section
- deleted Related information sections and old Jekyll headings
 
I need the following before this PR is accepted:

- [x] Technical review
- [x] Resolve question about base URL
- [x] Copy edit

whatsnew
Improve instructions for [configuring multi-site URLs](https://devdocs.magento.com/guides/v2.2/cloud/project/project-multi-sites.html) with Magento Commerce Cloud.

### CODE REVIEW

**Comment by hguthrie** [comment]
OKay, I updated according to technical feedback from @mveeramneni

File: guides/v2.1/cloud/project/project-multi-sites.md [hunk 1]
```diff
@@ -64,128 +59,145 @@ To configure routes in an integration environment:
...
 
-Make sure you name your websites, stores, and store views in your Cloud Admin the same as you did when you set up your local installation.
+1.  Access the public URLs:
 
-See [Set up multiple websites, stores, and store views in the Admin]({{ page.baseurl }}/config-guide/multi-site/ms_websites.html).
+    ```bash
+    magento-cloud environment:url
+    ```
 
-### Modify `magento-vars.php` {#cloud-multi-stores-magento-vars}
```
Comment (jeff-matthews) [inline guides/v2.1/cloud/project/project-multi-sites.md]: Did you delete all cross references to this heading?

File: guides/v2.1/cloud/project/project-conf-files_routes.md [hunk 2]
```diff
@@ -54,17 +54,18 @@ For example, if the project ID is `mswy7hzcuhcjw` on a branch called `refactorcs
...
+## Route options
 
 Configure each route separately using the following properties:
 
--  `type: upstream`—serves an application. Also, it has an `upstream` property that specifies the name of the application (as defined in `.magento.app.yaml`) followed by the `:http` endpoint.
--  `type: redirect`—redirects to another route. It is followed by the `to` property, which is an HTTP redirection to another route identified by its template.
--  `cache`—controls [caching for the route]({{ page.baseurl }}/cloud/project/project-routes-more-cache.html).
--  `redirects`—controls [redirect rules]({{ page.baseurl }}/cloud/project/project-routes-more-redir.html).
--  `ssi`—controls enabling of [Server Side Includes]({{ page.baseurl }}/cloud/project/project-routes-more-ssi.html).
+`type: upstream` | Serves an application. Also, it has an `upstream` property that specifies the name of the application (as defined in `.magento.app.yaml`) followed by the `:http` endpoint.
```
Comment (jeff-matthews) [inline guides/v2.1/cloud/project/project-conf-files_routes.md]: Using pipes to separate property from description in a list looks like a new style trend. I'm not opposed, but don't we normally use em dashes?

File: guides/v2.1/cloud/project/project-conf-files_routes.md [hunk 3]
```diff
@@ -104,29 +105,31 @@ This functions as a catch-all domain in a live environment.
 
 ### Routing a non-mapped domain
 
-You can route to a system that is not mapped to a domain using the dot (\.) to separate the subdomain.
+You can route to a system that is not mapped to a domain using the dot (\.) to separate the subdomain. For example, a project with an `add-theme` branch routes to `http://add-theme-projectID.us.magento.com/`.
```
Comment (jeff-matthews) [inline guides/v2.1/cloud/project/project-conf-files_routes.md]: using "a" dot versus "the" dot

File: guides/v2.1/cloud/project/project-multi-sites.md [hunk 4]
```diff
@@ -14,40 +10,39 @@ functional_areas:
...
-
-After successfully creating and testing multiple stores locally, you must:
+After successfully creating and testing the local installation to use multiple stores, you must prepare your Integration environment:
 
-1.	[Configure routes](#cloud-multi-stores-routes), which specify how incoming URLs are handled by {{site.data.var.ee}}.
-2.	[Set up websites, stores, and store views](#cloud-multi-stores-admin) in your {{site.data.var.ee}} server's {% glossarytooltip 29ddb393-ca22-4df9-a8d4-0024d75739b1 %}Admin{% endglossarytooltip %}.
-3.	[Modify `magento-vars.php`](#cloud-multi-stores-magento-vars) to specify the values of the `MAGE_RUN_TYPE` and `MAGE_RUN_CODE` variables.
-4.	[Deploy](#cloud-multi-stores-deploy) to your Integration branch and test.
+1.  **Configure routes**—specify how incoming URLs are handled by {{site.data.var.ee}}
+1.  **Set up websites, stores, and store views**—configure using your {{site.data.var.ee}} server Admin
```
Comment (jeff-matthews) [inline guides/v2.1/cloud/project/project-multi-sites.md]: server Admin? i don't think i've seen the Admin app referred to as that before. I've seen "Admin panel" and "Admin". We should have something in our style guide that dictates how we refer to the Admin UI.

File: guides/v2.1/cloud/project/project-multi-sites.md [hunk 5]
```diff
@@ -64,128 +59,145 @@ To configure routes in an integration environment:
...
-### Set up websites, stores, and store views {#cloud-multi-stores-admin}
 
-Set up in your {{site.data.var.ee}} Admin websites, stores, and store views identical to the ones you set up on your local system.
+1.  Save your changes to the `routes.yaml` file.
 
-#### Get your access information
+### Set up websites, stores, and store views
 
-To get the access information you need to log in to the Magento Admin:
+Set up your {{site.data.var.ee}} Admin websites, stores, and store views identical to the ones you set up on your local workstation.
```
Comment (jeff-matthews) [inline guides/v2.1/cloud/project/project-multi-sites.md]: This seems a little misleading... you set theses things up _using_ the Admin panel.

File: guides/v2.1/cloud/project/project-multi-sites.md [hunk 6]
```diff
@@ -64,128 +59,145 @@ To configure routes in an integration environment:
...
+    magento-cloud variable:list
+    ```
 
-1.	Open `magento-vars.php` in a text editor.
-2.	Uncomment everything after the first two lines.
-3.	Move the entire block starting with `if (isHttpHost("example.com")` after `function isHttpHost($host)`.
+Make sure you name your websites, stores, and store views in your Cloud Admin the same as you did when you set up your local installation.
 
-	Following is what the file should look like so far:
+See [Set up multiple websites, stores, and store views in the Admin]({{ page.baseurl }}/config-guide/multi-site/ms_websites.html).
```
Comment (jeff-matthews) [inline guides/v2.1/cloud/project/project-multi-sites.md]: This seems like the best way to refer to Admin.

File: guides/v2.1/config-guide/multi-site/ms_over.md [hunk 7]
```diff
@@ -16,57 +12,43 @@ functional_areas:
...
+-  Categories
+-  Products
+-  Currencies
 
-*   Default languages
-*   {% glossarytooltip 41aee03b-a5d5-49c2-8839-894090ef4e86 %}Domain{% endglossarytooltip %} names
-*   Categories
-*   Products
-*   Currency
+This flexible solution enables one Magento codebase and {% glossarytooltip 18b930cf-09cc-47c9-a5e5-905f86c43f81 %}Magento Admin{% endglossarytooltip %} to administer and display different stores. You configure the websites, stores, and store views in the Magento {% glossarytooltip 29ddb393-ca22-4df9-a8d4-0024d75739b1 %}Admin{% endglossarytooltip %}. Use certain variables in virtual hosts to start the Magento application using these websites or store views.
```
Comment (jeff-matthews) [inline guides/v2.1/config-guide/multi-site/ms_over.md]: I don't think it's necessary to use a glossary tooltip for a term twice in a paragraph. One time is sufficient.

File: guides/v2.1/config-guide/multi-site/ms_over.md [hunk 8]
```diff
@@ -16,57 +12,43 @@ functional_areas:
...
+A typical use is to set up stores with different options in different domains. For example, you could have one set of categories and products on one domain and another set of categories and products on a separate domain in a different language.
 
-You configure the websites, stores, and store views in the Magento {% glossarytooltip 29ddb393-ca22-4df9-a8d4-0024d75739b1 %}Admin{% endglossarytooltip %}. Use the `MAGE_RUN_TYPE` and `MAGE_RUN_CODE` variables in virtual hosts to start the Magento application using these websites or store views. 
+Consider the following terms:
 
-A typical use of `MAGE_RUN_TYPE` and `MAGE_RUN_CODE` is to set up stores with different options in different domains. For example, you could have one set of categories and products on one domain and another domain that has categories and products for a different language.
+-   **Website** is the top-level container for sites, shipping methods, payment methods, and so on. To create completely separate sites that do not share cart, shipping methods, and so on,  you must create separate websites.
```
Comment (jeff-matthews) [inline guides/v2.1/config-guide/multi-site/ms_over.md]: I'm not a fan of the term "and so on". It's unnecessary.

File: guides/v2.1/config-guide/multi-site/ms_over.md [hunk 9]
```diff
@@ -16,57 +12,43 @@ functional_areas:
...
+    Store views change the way pages are presented, and are typically used to display a store with different layouts or languages. You can manage different currencies per store view.
 
-* *Store* is contained by a {% glossarytooltip a3c8f20f-b067-414e-9781-06378c193155 %}website{% endglossarytooltip %}. In turn, a store contains at least one *store view*. 
+Each website and each {% glossarytooltip ca5a9ff1-8182-4fc4-a34b-9b3f831dbf3f %}store view{% endglossarytooltip %} must have a unique identifier. This identifier is required to use the `MAGE_RUN_TYPE` and `MAGE_RUN_CODE` variables as follows:
 
-  Multiple stores can share cart, user sessions, payment gateways, and so on, but they have separate catalog structures and Catalog price.
+-   `MAGE_RUN_TYPE` can be either `store` or `website`
 
-  Catalog Qty (inventory) can't be managed at the store level. Inventory is managed at Website or Global level only.
+    * Use `website` to load a website in your storefront.
```
Comment (jeff-matthews) [inline guides/v2.1/config-guide/multi-site/ms_over.md]: Lists with mixed terminal punctuation always stand out like a sore thumb to me

File: guides/v2.1/config-guide/multi-site/ms_websites.md [hunk 10]
```diff
@@ -113,41 +102,38 @@ Creating a root category is optional, but we show how to do it in this tutorial
...
+## Step 5: Change the website base URL
 
 To access a website using a unique {% glossarytooltip a05c59d3-77b9-47d0-92a1-2cbffe3f8622 %}URL{% endglossarytooltip %} like `http://french.magento.mg`, you must change the base URL for each site in the {% glossarytooltip 18b930cf-09cc-47c9-a5e5-905f86c43f81 %}Magento Admin{% endglossarytooltip %}.
 
-{% collapsible To change the website's base URL: %}
+{% collapsible To change the website base URL: %}
 
-1.	In the Admin, click **Stores** > Settings > **Configuration** > General > **Web**.
-2.	From the **Store View** list at the top of the page, click the name of one of your websites as the following figure shows.
+1.  In the Admin, click **Stores** > Settings > **Configuration** > General > **Web**.
```
Comment (jeff-matthews) [inline guides/v2.1/config-guide/multi-site/ms_websites.md]: Bold "General"

File: guides/v2.1/config-guide/multi-site/ms_websites.md [hunk 11]
```diff
@@ -157,32 +143,30 @@ For more information, see the [Magento User Guide](http://docs.magento.com/m2/ce
...
 
-You must perform this step last because after it's done, you'll lose access to the Magento Admin; your access returns after you set up virtual hosts as discussed in the web-server-specific topics.
+You must perform this step last because you will lose access to the Magento Admin; your access returns after you set up virtual hosts as discussed in the web-server-specific topics.
 
-{% collapsible To change the default store view's base URL: %}
+{% collapsible To change the default store view base URL: %}
 
-1.	In the Admin, click **Stores** > Settings > **Configuration** > General > **Web**.
-2.	From the **Store View** list at the top of the page, click **Default Config** as the following figure shows.
+1.  In the Admin, click **Stores** > Settings > **Configuration** > General > **Web**.
```
Comment (jeff-matthews) [inline guides/v2.1/config-guide/multi-site/ms_websites.md]: Bold "Settings" and "General"

File: guides/v2.1/cloud/project/project-conf-files_routes.md (hunk 2 above)
Comment (jeff-matthews) [reply guides/v2.1/cloud/project/project-conf-files_routes.md]: Oops! I didn't even think about it being a table. If it renders as a table, then please disregard.

File: guides/v2.1/cloud/project/project-multi-sites.md (hunk 4 above)
Comment (hguthrie) [reply guides/v2.1/cloud/project/project-multi-sites.md]: Oops! Slip-o-the-lazy...

File: guides/v2.1/config-guide/multi-site/ms_websites.md [hunk 12]
```diff
@@ -127,15 +127,15 @@ To access a website using a unique {% glossarytooltip a05c59d3-77b9-47d0-92a1-2c
...
+Magento gives you the option to add the store code to the site base URL, which simplifies the process of setting up multiple stores. Using this option, you do not have to create directories on the Magento file system to store `index.php` and `.htaccess`.
 
 This prevents `index.php` and `.htaccess` from getting out of sync with the Magento codebase in future upgrades.
 
 For more information, see the [Magento User Guide](http://docs.magento.com/m2/ce/user_guide/stores/store-urls.html){:target="_blank"}.
 
 {% collapsible To add the store code to the base URL: %}
 
-1.	In the Admin, click **Stores** > Settings > **Configuration** > General > **Web**.
+1.	In the _Admin_ panel, click **Stores** > Settings > **Configuration** > **General** > **Web**.
```
Comment (jeff-matthews) [inline guides/v2.1/config-guide/multi-site/ms_websites.md]: Bold  "Settings"

File: guides/v2.1/config-guide/multi-site/ms_websites.md [hunk 13]
```diff
@@ -155,24 +155,24 @@ You must perform this step last because you will lose access to the Magento Admi
 
 {% collapsible To change the default store view base URL: %}
 
-1.  In the Admin, click **Stores** > Settings > **Configuration** > General > **Web**.
+1.  In the _Admin_ panel, click **Stores** > Settings > **Configuration** > General > **Web**.
```
Comment (jeff-matthews) [inline guides/v2.1/config-guide/multi-site/ms_websites.md]: Bold "Settings" and "General"

**Pull request merged**